### PR TITLE
 step 1.26.0-next -> 1.27.0

### PR DESCRIPTION
**Pull request by AlvaroVega**
Mimics PR https://github.com/telefonicaid/perseo-fe/pull/709

### CODE REVIEW

File: rpm/SPECS/cep.spec [hunk 1]
```diff
@@ -164,6 +164,13 @@ rm -rf $RPM_BUILD_ROOT
 %{_install_dir}
 
 %changelog
+* Wed May 17 2023 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 1.27.0
+- Add support to run tests with node 18
+- Set Nodejs 16 as minimum version in packages.json (effectively removing Nodev12 and Nodev14 from supported versions)
+- Upgrade uuid dep from ~1.4.2 to 9.0.0
+- Fix: do not flat in to map object or array attributes of event (#720)
+- Remove unnecessary event data transformations (stripped, location parse, datetime parse) (#706, #712)
```
Comment (fgalan) [inline rpm/SPECS/cep.spec:172]:
```suggestion
- Add: support to run tests with node 18
- Fix: do not flat in to map object or array attributes of event (#720)
- Remove: unnecessary event data transformations (stripped, location parse, datetime parse) (#706, #712)
- Upgrade uuid dep from ~1.4.2 to 9.0.0
- Set Nodejs 16 as minimum version in packages.json (effectively removing Nodev12 and Nodev14 from supported versions)
```

**Comment by AlvaroVega** [comment]
waiting for https://jenkins.urbo2.es/job/urbo2-devel-pqa-test-daily/845/

**Comment by AlvaroVega** [comment]
> waiting for https://jenkins.urbo2.es/job/urbo2-devel-pqa-test-daily/845/

Already end. CI about perseo was fine.

File: rpm/SPECS/cep.spec [hunk 2]
```diff
@@ -164,6 +164,13 @@ rm -rf $RPM_BUILD_ROOT
 %{_install_dir}
 
 %changelog
+* Wed May 17 2023 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 1.27.0
```
Comment (fgalan) [inline rpm/SPECS/cep.spec:167]:
```suggestion
* Thu May 18 2023 Alvaro Vega <alvaro.vegagarcia@telefonica.com> 1.27.0
```

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**